test/collection_test.js

```diff
@@ -995,6 +1036,40 @@ describe("Collection", () => {
         });
       });
 
+      describe("incoming changes hook", () => {
+        it("should be called", () => {
+          let hookCalled = false;
+          articles = testCollection({
+            hooks: {
+              "incoming-changes": [
+                function(payload) {
+                  hookCalled = true;
+                  return payload;
+                }
+              ]
+            }
+          });
+
+          return articles.pullChanges(result)
+            .then(_ => {expect(hookCalled).to.be.true;});
```

Comment:
Nit: unnecessary curly braces. Also, always prefer `eql(true)` for boolean asserts.